dawn/prototype/mylibVerticalSumDriver.cpp

```diff
@@ -0,0 +1,27 @@
+#include "mylib_interface.hpp"
+#include <fstream>
+#include <gridtools/clang_dsl.hpp>
+
+#include "generated_verticalSum.hpp"
+
+using namespace mylibInterface;
```

Comment:
don't use `using namespace mylibInterface` in the tests because we must not rely on the user doing this and we need to make sure that ADL works properly.